src/pages/settings/Wallet/ExpensifyCardPage.js

```diff
@@ -1,5 +1,5 @@
 import PropTypes from 'prop-types';
-import React, {useState} from 'react';
+import React, {useReducer} from 'react';
```

Comment:
I don't think we should start with `useReducer`. We have avoided it until now and it looks like the incorrect pattern for what we are trying to achieve here.